crt_portal/static/sass/_uswds-theme-custom-styles.scss

```diff
@@ -258,6 +261,40 @@ form#report-form {
   }
 }
 
+.tr-status-new {
+  td {
+    font-weight: bold;
+  }
+}
+
+// crt view statuses
+.status-new {
+  color: $white !important;
+  background-color: $blue-warm-vivid-70;
+  padding: 4px;
+  padding-left: 10px;
+  padding-right: 10px;
+  border-radius: 8.5rem;
+}
+
+.status-open {
+  background-color: $gold-20v;
+  font-weight: bold;
+  padding: 4px;
+  padding-left: 10px;
```

Comment:
Our spacing values are generally in increments of `8px` (`0.5rem`), but maybe this is an exception in the annotated mockup?